mount/lookup_unix.go

```diff
@@ -20,6 +24,9 @@ func Lookup(dir string) (Info, error) {
 	if err != nil {
 		return Info{}, err
 	}
+
+	// Sort descending order by Info.Mountpoint
+	sort.Sort(sort.Reverse(InfoSlice(mounts)))
```

Comment:
You can use https://golang.org/pkg/sort/#Slice instead of adding a new type.

Reply:
@mlaventure Yes, it's fine, will do :slightly_smiling_face:

Reply:
Done.